tests/compile/test_builders.py

```diff
@@ -180,8 +194,9 @@ def go(inps, gs):
         dedz = vector("dedz")
         op_mul_grad = cls_ofg([x, y, dedz], go([x, y], [dedz]))
 
-        op_mul = cls_ofg([x, y], [x * y], grad_overrides=go)
-        op_mul2 = cls_ofg([x, y], [x * y], grad_overrides=op_mul_grad)
+        with pytest.warns(FutureWarning, match="grad_overrides is deprecated"):
```

Comment:
Just use `lop_overrides` instead of adding these warning checks?

Reply:
This way I am sure I am not accidentally breaking the old API, that can be done when we remove the old API, at which point we also need to assess how much redundancy there is between current grand and lop overrides tests